reference-sample-config/collection-save.json

```diff
@@ -0,0 +1,219 @@
+					"editable": true,
+					"index": 6,
+					"inputType": "select",
+					"label": "Board",
+					"name": "Board",
+					"placeholder": "Select Board",
+					"renderingHints": {},
+					"required": true,
+					"visible": true
+				}, {
+					"code": "medium",
+					"dataType": "list",
+					"description": "",
+					"editable": true,
+					"index": 7,
+					"inputType": "select",
+					"label": "Medium of Instruction",
+					"name": "medium",
+					"placeholder": "Select Medium",
+					"renderingHints": {},
+					"required": true,
+					"visible": true
+				}, {
+					"code": "gradeLevel",
+					"dataType": "list",
+					"description": "Class",
+					"editable": true,
+					"index": 8,
+					"inputType": "multiselect",
+					"label": "Question Paper classes",
+					"name": "Class",
+					"placeholder": "Select Class",
+					"renderingHints": {},
+					"required": true,
+					"visible": true
+				}, {
+					"code": "subject",
+					"dataType": "list",
+					"description": "",
+					"editable": true,
+					"index": 9,
+					"inputType": "select",
+					"label": "Question Paper subjects",
+					"name": "Subject",
+					"placeholder": "Select Subject",
+					"renderingHints": {},
+					"required": true,
+					"visible": true
+				}, {
+					"code": "topic",
+					"dataType": "list",
+					"description": "Choose a Topics",
+					"editable": true,
+					"index": 10,
+					"inputType": "topicselector",
+					"label": "Topics Covered",
+					"name": "Topic",
+					"placeholder": "Choose Topics",
+					"renderingHints": {},
+					"required": false,
+					"visible": true
+				}, {
+					"code": "audience",
+					"dataType": "list",
+					"description": "",
+					"editable": true,
+					"index": 11,
+					"inputType": "select",
+					"label": "Audience Type",
+					"name": "Audience",
+					"placeholder": "Select Audience",
+					"renderingHints": {},
+					"range": ["Instructor", "Learner"],
+					"required": false,
+					"visible": true
+				}, {
+					"code": "author",
+					"visible": true,
+					"editable": true,
+					"dataType": "text",
+					"renderingHints": {},
+					"name": "Author",
+					"description": "Original Author",
+					"index": 12,
+					"inputType": "text",
+					"label": "Original Author",
+					"placeholder": "Author",
+					"required": false
+				}, {
+					"code": "attributions",
+					"visible": true,
+					"editable": true,
+					"dataType": "list",
+					"renderingHints": {},
+					"name": "attribution",
+					"description": "Attributions",
+					"index": 13,
+					"inputType": "text",
+					"label": "Attributions",
+					"placeholder": "Attributions",
+					"required": false
+				}, {
+					"code": "copyright",
+					"visible": true,
+					"editable": true,
+					"dataType": "text",
+					"renderingHints": {},
+					"name": "Copyright",
+					"description": "Copyright",
+					"index": 14,
+					"inputType": "text",
+					"label": "Copyright",
+					"placeholder": "Copyright",
+					"required": false
+				}]
```

Comment:
@bharatkashyap Are you reading the correct configuration to start with? You seem be missing - License and Year of creation. See below
<img width="1400" alt="image" src="https://user-images.githubusercontent.com/10580754/114746938-e5f58100-9d6d-11eb-9ae4-51f108d4cd3a.png">

Reply:
I've added @surendrasinghs - This was missing on dev. Have picked up from the form on production.